logql/v2/parser_test.go

```diff
@@ -440,7 +480,7 @@ func TestParseExpr(t *testing.T) {
 		},
 		// log query expressions with format expressions
 		{
-			input: `{app="first"} |= "value" | json | line_format "loop{{ .first }}blop {{.status_code}}" | label_format first=value,status_code="blop{{.value}}"`, //nolint:lll
+			input: `{app="first"} |= "value" | json | line_format "loop{{ .first }}blop {{.status_code}}" | label_format first="value",status_code="blop{{.value}}"`, //nolint:lll
```

Comment:
Per [LogQL docs](https://grafana.com/docs/loki/latest/logql/log_queries/#labels-format-expression) there's a difference between `first=value` and `first="value"`, the former referring to a `value` label and the latter producing `value` as literal value.